### PR TITLE
New guides - won't be visible until new structure is implemented

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: guides/test-debugging-1.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+<Frame caption="Test report run with passed and failed active tests, 12/2024">
+  <img
+    src="/images/guides/test-report.png"
+    alt="Test report run with passed and failed tests"
+  />
+</Frame>
+
+There are some other edge cases why tests might fail. Why have listed here the most common ones here.
+
+Now, you can start debugging your tests. [This is how.](guides/test-debugging-2)
```
Comment (sker65) [inline guides/test-debugging-1.mdx:123]: Link does not work try /guides/test-debugging-2 maybe?

File: guides/test-debugging-2.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  <img
+    src="/images/guides/detailed-prompt.png"
+    alt="an example of a detailed prompt in Octomind e2e tests"
+    width="300"
+  />
+</Frame>
+
+Your Octomind-specific prompting skills will improve with time, but if you ever get stuck, [reach out to our team on Discord](https://discord.gg/3ShnZMKRfA) for guidance.
+
+Okay, what if the prompt is not the cause of the problem? This is how you [fix the test using our visual editor](guides/test-debugging-3).
```
Comment (sker65) [inline guides/test-debugging-2.mdx:78]: Link does not work try /guides/test-debugging-3 maybe?

**Review by sker65 — APPROVED** [review]
